workload.yaml

```diff
@@ -1,11 +1,14 @@
 ---
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
   name: workload-certificate
   namespace: default
 spec:
   secretName: workload-tls
+  subject:
+    organizations:
+      - Example Organization
   dnsNames:
     - workload.demo.example.com
   issuerRef:
```

Comment:
nit: There's also a much newer nginx image that we could use in the deployment below: 1.19.3